ladybug_comfort/di.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+"""Utility function for calculating Discomfort Index (DI)."""
+
+
+def discomfort_index(ta, rh):
+    """Calculate discomfort index (DI) from air temperature and relative
+    humidity.
+
+    Discomfort Index is derived from original work carried out by Eral C. Thom
+    [1] which defined discomfort index based on dry-bulb and wet-bulb
+    temperature.
+
+    Note:
+        [1]  Thom, E.C. (1959) "The Discomfort Index". Weatherwise, 12, 57-61.
+    Args:
```

Comment:
Can you add a line break above `Args`? The docstring might not render correctly in our [online docs](https://www.ladybug.tools/ladybug-comfort/docs/ladybug_comfort.html) otherwise.